api/hooks/apiStatusHook/index.js

```diff
@@ -0,0 +1,17 @@
+module.exports = function apiStatusHook() {
+  ////////////////
+  // API Status //
+  ////////////////
+  return {
+    getStatus: function () {
+      return new Promise(function (resolve, reject) {
+        Game.count().exec(function (error, game) {
```

Comment:
Not that I think anyone would do this, but as it stands it's possible to ddos the server by pinging this endpoint repeatedly. I think it might be worth caching the value for like 3-5 minutes at a time or something similar. I'm not sure the best way to do that since we don't really have a cache wrapper. But maybe in pro scenarios have it use the memcached server or something? cc @itsalaidbacklife

Reply:
interesting point. That said, is this any more vulnerable than any other endpoint?

Reply:
Not really but it seems a little easier to automate a hit against. I'm not really bothered either way, just something I see as a potential problem.

Reply:
I agree with the DDOS but can share that is not just this endpoint, it would be all of them. I think you could require authentication for them so at least you could track who is doing this and limit or ban them, for example. That assuming you are monitoring the service. But there will be always endpoints that are meant to be public, like the sign up endpoint. 

I was reading that Sails supports [using Express middlewares](https://sailsjs.com/documentation/concepts/middleware#?http-middleware) and these might be a quick solution. Check this [blogpost](https://dev.to/brunohgv/limiting-node-js-api-calls-with-express-rate-limit-11kl) for guidance. Could be tackled in a separate PR.

Reply:
Yeah by no means a blocker, excellent discussion on everything— really appreciate the level of detail and research :)